src/FsAutoComplete/CodeFixes/AddMissingXmlDocumentation.fs

```diff
@@ -115,7 +115,7 @@ let fix (getParseResultsForFile: GetParseResultsForFile) : CodeFix =
       let commentsAndPos = tryGetCommentsAndSymbolPos parseAndCheck.GetAST fcsPos
 
       match commentsAndPos with
-      | Some(docLines, docRange, symbolPos) ->
+      | Some(docLines, docRange, symbolPos, isAutoProperty) ->
         let lineStrOfSymbol = _sourceText.GetLine symbolPos |> Option.defaultValue ""
         let signatureData = parseAndCheck.TryGetSignatureData symbolPos lineStrOfSymbol
```

Comment:
The signatureData for the setter was returned here.
This does indeed have a member parameter.